examples/basic/variables.tf

```diff
@@ -31,3 +27,13 @@ variable "resource_tags" {
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
+variable "kms_key_crn" {
```

Comment:
not used by basic example

Reply:
they are actually used by the basic example, but yes I can remove them.